__tests__/app.test.js

```diff
@@ -264,6 +261,81 @@ describe("GET/non-existent API", () => {
     });
   })
 
+  describe("PATCH/api/articles/:article_id", () => {
+    test("200: test that when given a patch request with an article_id and a body containing a number of votes that the article's votes will be updated", () => {
+      return request(app)
+      .patch("/api/articles/1")
+      .send({inc_votes : 50})
+      .expect(200)
+      .then(({body}) => {
+        const {article} = body
+        expect(article.votes).toEqual(150)
+      }).then(() => {
+        return request(app)
+        .patch("/api/articles/1")
+        .send({inc_votes : -50})
+        .expect(200)
+        //re-setting the votes after each test so that test data isn't affected- not sure if this is good practice
+      })
+    })
+    test("200: test that when the result is a negative number that the article will be returned", () => {
+      return request(app)
+      .patch("/api/articles/1")
+      .send({inc_votes : -300})
+      .expect(200)
+      .then(({body}) => {
+        const {article} = body
+        expect(article.votes).toEqual(-200)
+      }).then(() => {
+        return request(app)
+        .patch("/api/articles/1")
+        .send({inc_votes : 300})
+        .expect(200)
+      })
```

Comment:
Ironically, you needed this functionality to do the 'reset' one lines 273-278 

Ideally get rid of the reset and your testing will make sense again